apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -49,14 +47,15 @@ class GqlWithKnexLoadList {
         return all
     }
 
-    async loadChunk (skip = 0) {
```

Comment:
this method already used in many places. are you sure that you want to change their signature? (optional)